open-build-service-api/src/lib.rs

```diff
@@ -949,6 +984,23 @@ impl<'a> ProjectBuilder<'a> {
             .map(|e| e.name)
             .collect())
     }
+
+    pub async fn rebuild_all(&self, filters: &RebuildFilters) -> Result<()> {
```

Comment:
> Not rebuild_all that were in the filter..

If you pass an empty filter, it indeed rebuilds everything, which matches OBS's own behavior here. (Whether or not it's *good* behavior is up for debate, but I was hesitant to stray too far from it.)

It's indeed pretty limited right now, it's just also a bit harder for me to test the way rebuilds work in other cases, and it wasn't as high-priority since the runner never needs to do that.

Reply:
Why not just call it rebuild in that case? Also i'm a bit wary of accidentally leaving filters empty. But that's a different discussion